### PR TITLE
Fix required field for ChartRepository struct in Piped config reference

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/docs-v0.22.x/operator-manual/piped/configuration-reference.md [hunk 1]
```diff
@@ -24,8 +24,7 @@ spec:
 | pipedKeyFile | string | The path to the file containing the generated key string for this piped. | Yes |
 | pipedKeyData | string | Base64 encoded string of Piped key. Either pipedKeyFile or pipedKeyData must be set. | Yes |
 | apiAddress | string | The address used to connect to the control-plane's API. | Yes |
-| syncInterval | duration | How often to check whether an application should be synced. Default is `1m`. | No |
-<!-- TODO: Add doc for appConfigSyncInterval -->
+| syncInterval | duration | How often to check whether an application should be synced. Default is `1m`. | No | <!-- TODO: Add doc for appConfigSyncInterval -->
```
Comment (nghialv) [inline docs/content/en/docs-v0.22.x/operator-manual/piped/configuration-reference.md:27]: To fix the broken layout of the table.
https://pipecd.dev/docs-dev/operator-manual/piped/configuration-reference/#piped-configuration

Comment (nakabonne) [reply docs/content/en/docs-v0.22.x/operator-manual/piped/configuration-reference.md:27]: Oops. Thanks

File: docs/content/en/docs/operator-manual/piped/configuration-reference.md [hunk 2]
```diff
@@ -60,12 +59,12 @@ spec:
 | Field | Type | Description | Required |
 |-|-|-|-|
 | type | string | The repository type. Currently, HTTP and GIT are supported. Default is HTTP. | No |
-| name | string | The name of the Helm chart repository. Note that is not a Git repository but a [Helm chart repository](https://helm.sh/docs/topics/chart_repository/). | Yes |
-| address | string | The address to the Helm chart repository. | Yes |
+| name | string | The name of the Helm chart repository. Note that is not a Git repository but a [Helm chart repository](https://helm.sh/docs/topics/chart_repository/). | Yes if type is HTTP |
```
Comment (nakabonne) [inline docs/content/en/docs/operator-manual/piped/configuration-reference.md:62]: Nice. Either is fine with me but I would love to unify how to write it when this field is different depending on the condition: https://pipecd.dev/docs-dev/user-guide/configuration-reference/#analysismetrics

Comment (nakabonne) [reply docs/content/en/docs/operator-manual/piped/configuration-reference.md:62]: Yours looks simpler so I will be fine with this.

**Comment by nakabonne** [comment]
Well, I'm still confused. Do we have to update ones under `docs-dev/`?

**Comment by nghialv** [comment]
Nice catch! Let me update that one.

**Comment by nghialv** [comment]
Updated!

**Comment by nakabonne** [comment]
Nice
/lgtm

**Pull request merged**